### DISCUSSION TITLE
Problem: Offset of the map shifting.

### DISCUSSION THREAD

**Discussion by NightmareDay** [ppy/osu#30692] [Q&A]
Sometimes, after the screen is minimized, the offset of the map shifts. 
https://github.com/user-attachments/assets/ccc09d5f-fbfd-4692-b1db-afa5d7aebdbe
Sometimes the offset changes so much that the sound of the note does not match at all.

I found a solution by changing the playback device, but it's such is a bad pleasure...
(in osu!stable this didn't happen)

**Comment by peppy** [comment]
We're still tracking ongoing audio issues with certain device setups. I can't find the discussion thread right now, but there are a few solutions offered around the place. You could [have a search yourself](https://github.com/ppy/osu/discussions?discussions_q=label%3Aaudio).

**Comment by peppy** [comment]
If you're still encountering this, please read https://github.com/ppy/osu/issues/31843 and test the provided fix if you're able!
